tests/helpers/basic_models.py

```diff
@@ -349,6 +349,21 @@ def forward(self, x1: Tensor, x2: Tensor, x3: Tensor, scale: int):
         return self.model(scale * (x1 + x2 + x3))
 
 
+class BasicModel_ConvNet_Optim(nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.layer = nn.Conv2d(3, 2, 1, bias=True)
+        # Initialize weights and biases for
+        # easy reproducibility
+        self.layer.weight.data.fill_(0.1)
+        if self.layer.bias is not None:
+            self.layer.bias.data.fill_(1)
```

Comment:
Added none-check here because adding `-> None` typehint to the `__init__` function apparently turns `self.layer.bias` to `Optional[Tensor]`, which triggers this `mypy` error.

> tests/helpers/basic_models.py:359: error: Item "None" of "Optional[Tensor]" has no attribute "data"

Reply:
@greentfrapp Does casting resolve the issue? I came across a few `mypy` issues myself that were solved with casting the variable to the correct type.

Reply:
That is, assuming all parameters are tensors?